api/app/Extensions/JWTAuth/JWTAuth.php

```diff
@@ -0,0 +1,58 @@
+<?php namespace App\Extensions\JWTAuth;
+
+use Illuminate\Http\Request;
+use Tymon\JWTAuth\Token;
+use Tymon\JWTAuth\JWTAuth as JWTAuthBase;
+use RuntimeException;
+use App\Exceptions\BadRequestException;
+use App\Exceptions\UnauthorizedException;
+use App\Exceptions\TokenInvalidException;
+use App\Exceptions\UnprocessableEntityException;
+use Tymon\JWTAuth\Exceptions\TokenExpiredException;
+
+class JWTAuth extends JWTAuthBase
+{
+    /**
+     * Get the token contained within the current request.
+     *
+     * @param  Request  $request
+     * @throws BadRequestException
+     * @return Token
+     */
+    public function getTokenFromRequest(Request $request)
+    {
+        if (!$token = $this->setRequest($request)->getToken()) {
+            throw new BadRequestException('Token not provided.');
+        }
+
+        return $token;
+    }
+
+    /**
+     * Get the user from the request token.
+     *
+     * @param  Token  $token
+     * @throws UnauthorizedException
+     * @throws UnprocessableEntityException
+     * @throws RuntimeException
+     * @return \App\Model\User
+     */
+    public function getUser(Request $request)
```

Comment:
@Redjik yea I specifically requested that this method be able to just be `$this->jwtAuth->getUser($request)` as *many* controllers will want the method out of the request token, so it makes sense for it to be a simple one liner.